scripts/build

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Stop on any error
+set -e
+
+script_dirirectory="$( cd "$( dirname "$0" )" && pwd )"
+project_dirirectory=$script_dirirectory/..
+
+# Specify the Docker image name
+image_name='mgmt/mgmt'
+
+# Build the image which contains the compiled binary
+docker build -t $image_name \
+             --file=$project_dirirectory/docker/Dockerfile $project_dirirectory
+
+# Remove the container if it already exists
+docker rm -f mgmt-export 2> /dev/null
+
+# Start the container in background so we can "copy out" the binary
+docker run -d --name=mgmt-export $image_name bash -c 'while true; sleep 1000; done'
+
+# Remove the current binary
+rm $project_dirirectory/app
+
+# Get the binary from the container
+docker cp mgmt-export:/go/src/app/app $project_dirirectory/app
```

Comment:
When I run `make build` it creates the `app` file. I can rename it to `mgmt`. What do you think?

If you mean the directory inside the container (`/go/src/app/app`), then it really doesn't make much difference how you call it. I took the name from [here](https://github.com/docker-library/golang/blob/ce284e14cdee73fbaa8fb680011a812f272eae2e/1.6/onbuild/Dockerfile#L4).